test/Google.Api.Gax.Tests/FakeSchedulerTest.cs

```diff
@@ -15,26 +15,15 @@ namespace Google.Api.Gax.Tests
 {
     public class FakeSchedulerTest
     {
+        // Note: no tests for cancellation at the moment as they're fiendishly difficult to write in a robust way.
+        // We need to revisit FakeScheduler yet again...
+
         [Fact]
         public async Task SimpleDelay()
         {
             var scheduler = new FakeScheduler();
             await scheduler.RunAsync(() => scheduler.Delay(TimeSpan.FromTicks(1000)));
             Assert.Equal(1000, scheduler.Clock.GetCurrentDateTimeUtc().Ticks);
         }
-
-        [Fact]
-        public async Task Cancellation()
```

Comment:
Would rather not, because the API for scheduling the cancellation is likely to change too. (My ScheduleCancellation didn't do exactly what I wanted.)